docker-compose.stage.yml

```diff
@@ -52,11 +52,15 @@ services:
       - TC_RESAMPLING_METHOD=nearest
       - TC_REPROJECTION_METHOD=nearest
       - TC_PNG_COMPRESS_LEVEL=1
+
   pixel-driller:
-    image: ghcr.io/nismod/jsrat-pixel_driller:canary
+    image: ghcr.io/nismod/jsrat-pixel-driller:canary
```

Comment:
This should be changed in #253 too.

Reply:
With respect to #253:
Change for the `name` and `tags` keys?
Is `context` the build folder (so stays with an underscore)?

Reply:
Yeah. `context` points to the location of the Dockerfile. `tags` sets the name that’s used for the package in GHCR: https://github.com/nismod/irv-jamaica/pkgs/container/jsrat-pixel_driller

I built that image by running the workflow manually, under the Actions tab, to test whether it works.